content/docs/tutorials/certificate-defaults/README.md

```diff
@@ -141,34 +153,34 @@ That means anyone can follow this tutorial even without their own domain.
 
 ## 1 - Defaulting optional fields
 
-In this section we will create rules which set 3 fields for all `Certificate` resources automatically.
-None of the 3 fields here are required fields, but they might need to be set depending on platform and issuer preferences.
+In this section we will create rules which set three fields for all `Certificate` resources automatically.
+None of the three fields here are required fields, but they might need to be set depending on platform and issuer preferences.
 These rules will:
 
 - Set a default value of: `revisionHistoryLimit: 2`.
-- Set a default value of `Always` under `spec.privateKey.rotationPolicy`.
+- Set a [default value of `Always` under `spec.privateKey.rotationPolicy`](../../usage/certificate/#the-rotationpolicy-setting).
 - Set defaults for all `spec.privateKey` fields.
 
-> ℹ️ Note how these rules tackle the first two of our [3 uses cases](#use-cases).
+> ℹ️ Note how these rules tackle the first two of our [uses cases](#use-cases).
 
 1. First take a look at the `ClusterPolicy`:
 
-    ```yaml file=../../../../public/docs/tutorials/certificate-defaults/cpol-mutate-certificate-defaults.yaml
+    ```yaml file=../../../../public/docs/tutorials/certificate-defaults/cpol-mutate-certificates-0.yaml
     ```
-    🔗 <a href="cpol-mutate-certificate-defaults.yaml">`cpol-mutate-certificate-defaults.yaml`</a>
+    🔗 <a href="cpol-mutate-certificates-0.yaml">`cpol-mutate-certificates-0.yaml`</a>
 
 1. Apply the policy to the cluster and check that it is ready:
 
     ```shell
-    kubectl apply -f cpol-mutate-certificate-defaults.yaml
+    kubectl apply -f cpol-mutate-certificates-0.yaml
     kubectl get cpol
     ```
     
     When the `ClusterPolicy` is ready the output should look like this:
 
     ```log
     NAME                            BACKGROUND   VALIDATE ACTION   READY   AGE   MESSAGE
```

Comment:
did this line really not change its formatting to match the next line?

Reply:
Just fixed a couple minor CI fails and I'll check the latest preview deploy.
I do need to redo the output examples anyway due to the condensing of policies into one file.